test/api/controllers/realtime.test.js

```diff
@@ -143,15 +141,13 @@ describe('RealtimeController', () => {
     it('should throw an error if body is not provided',() => {
       request.input.body = null;
 
-      should(() => {
-        realtimeController.count(request);
-      }).throw(BadRequestError, { id: 'api.assert.body_required' });
+      should(realtimeController.count(request))
+        .rejectedWith(BadRequestError, { id: 'api.assert.body_required' });
     });
 
     it('should throw an error if roomId is not provided',() => {
-      should(() => {
-        realtimeController.count(request);
-      }).throw(BadRequestError, { id: 'api.assert.missing_argument' });
+      should(realtimeController.count(request))
```

Comment:
must be either returned or awaited, otherwise test will always pass even if the condition isn't verified